base/s2i/assemble

```diff
@@ -3,61 +3,106 @@
 # Setup Environment Variables
 LOCAL_SOURCE_DIR="${LOCAL_SOURCE_DIR:-/tmp/src}"
 HYBRIS_HOME="${HYBRIS_HOME:-/opt/hybris}"
+HYBRIS_PLATFORM_FOLDER="${HYBRIS_PLATFORM_FOLDER:-hybris/bin/platform}"
 HYBRIS_SOURCE_FOLDER="${HYBRIS_SOURCE_DIR:-hybris}"
+HYBRIS_LOCAL_PROPERTIES_PATH="${HYBRIS_LOCAL_PROPERTIES_PATH-hybris/config/local.properties}"
+HYBRIS_SERVER_XML_PATH="${HYBRIS_SERVER_XML_PATH-hybris/config/tomcat/conf/server.xml}"
 HYBRIS_CONTENT_DIR="${LOCAL_SOURCE_DIR}/${HYBRIS_SOURCE_FOLDER}"
 ANT_BUILD_TASKS="${ANT_TASKS:-clean all}"
 ANT_DEPLOY_TASKS="${ANT_TASKS:-deploy}"
 ANT_INITIALIZE_TASKS="${ANT_TASKS:-initialize}"
+SOURCE_REPO_LOCAL_PROPERTIES_OVERRIDE="${SOURCE_REPO_LOCAL_PROPERTIES_OVERRIDE}"
+APACHE_JVM_ROUTE_NAME="${APACHE_JVM_ROUTE_NAME}"
+HYBRIS_ANT_TASK_NAMES="${HYBRIS_ANT_TASK_NAMES}"
 
 if [ ! -d "${HYBRIS_CONTENT_DIR}" ]; then
     echo "Could Not Locate Hybris Directory in Source!"
     exit 1
 fi
 
 # Copying Source to Hybris Server
-echo "---> Copying Hybris application source ..."
-cp -r ${HYBRIS_CONTENT_DIR}/* "${HYBRIS_HOME}"
+echo "---> Copying Hybris application source cp -r ${HYBRIS_CONTENT_DIR}/* ${HYBRIS_HOME}/hybris"
+cp -r ${HYBRIS_CONTENT_DIR}/* ${HYBRIS_HOME}/hybris
+
+#####Below section unitl the commented lines added on April 5 2021 for the Hybris Operator changes
+
+if [ ! -z "${SOURCE_REPO_LOCAL_PROPERTIES_OVERRIDE}" ]; then
+    echo "---> Hybris config  ${HYBRIS_HOME}/${HYBRIS_LOCAL_PROPERTIES_PATH} is replaced by ${SOURCE_REPO_LOCAL_PROPERTIES_OVERRIDE}"
+    curl -k -o ${HYBRIS_HOME}/${HYBRIS_LOCAL_PROPERTIES_PATH} ${SOURCE_REPO_LOCAL_PROPERTIES_OVERRIDE}
```

Comment:
please wrap the arguments in double quotes, otherwise unexpected happens

Reply:
Done